src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const dailyRate = pricePerDay * days;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const smallDiscountDays = 3;
+  const bigDiscountDays = 7;
+
+  if (days < smallDiscountDays) {
+    return dailyRate;
+  } else if (days >= smallDiscountDays && days < bigDiscountDays) {
+    return dailyRate - smallDiscount;
+  } else {
+    return dailyRate - bigDiscount;
+  }
```

Comment:
You can simplify this complex structure

1. Get rid of `else`. If there's `return` inside `if` block code would stop executing anyway on return
2. Avoid using double conditions. Choose direction 0...7 days or 7...0 days and follow it.

Check the checklist and code examples there once again.